### PR TITLE
Create using-breakout-rooms.md

### PR DESCRIPTION
**Pull request by ricardo-msft**
I'm trying to add a new page to the Microsoft Teams documentation for using Breakout Rooms, a new feature. Does not seem to exist anywhere yet.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/using-breakout-rooms.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Using Breakout Rooms in Teams
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:2]:
```suggestion
title: Using breakout rooms in Teams
```
```suggestion
title: Using Breakout Rooms in Teams
```

Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:2]:
```suggestion
title: Using breakout rooms in Teams
```

File: Teams/using-breakout-rooms.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ms.topic: article
+ms.service: msteams
+ms.collection: 
+  - M365-collaboration
+appliesto: 
+  - Microsoft Teams
+description: End Users can learn how to use Breakout Rooms in Microsoft Teams
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:9]:
```suggestion
description: End users can learn how to use breakout rooms in Microsoft Teams
```

File: Teams/using-breakout-rooms.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+# Using Breakout Rooms in Microsoft Teams
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:12]:
```suggestion
# Using breakout rooms in Microsoft Teams
```
```suggestion
# Using Breakout Rooms in Microsoft Teams
```

File: Teams/using-breakout-rooms.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+As a meeting organizer, you can create new breakout rooms at the start of a scheduled meeting.
+
+## Create breakout rooms
+
+Use these steps to create a breakout room once a meeting begins.
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:18]:
```suggestion
Use these steps to create a breakout room once a meeting begins:
```

File: Teams/using-breakout-rooms.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+1. In theTeams desktop client, select the Breakout rooms icon on the control bar.
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:20]:
```suggestion
1. In the Teams desktop client, select the breakout rooms icon on the control bar.
```

File: Teams/using-breakout-rooms.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+2. Use the dropdown menu to select the number of rooms you would like. You can create up to 50 breakout rooms in a single meeting.
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:22]:
```suggestion
2. Use the dropdown menu to select the number of rooms you would like to create. You can create up to 50 breakout rooms in a single meeting.
```

File: Teams/using-breakout-rooms.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+3. Choose the breakout room option you prefer:
+
+    - **Automatically:** Teams divides participants equally between the rooms.
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:26]:
```suggestion
    - **Automatically**: Teams divides participants equally between the rooms.
```

File: Teams/using-breakout-rooms.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+    - **Manually:** You can choose which room each participant is placed in.
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:27]:
```suggestion
    - **Manually**: You can choose which room each participant is placed in.
```

File: Teams/using-breakout-rooms.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+4. Select **Create Rooms**. The Breakout Rooms pane will open.
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:29]:
```suggestion
4. Select **Create Rooms**. The breakout rooms pane will open.
```

File: Teams/using-breakout-rooms.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+
+## Start breakout rooms
+
+When you are ready to start your breakout sessions, select **Start rooms.**
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:33]:
```suggestion
When you are ready to start your breakout sessions, select **Start rooms**.
```

File: Teams/using-breakout-rooms.md [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+
+You will see participants leaving the main meeting and joining their breakout rooms.
+
+If a participant leaves their breakout room the organizer can move the participant back to the breakout room by selecting the ellipses […] next to the participant’s name and select Ask to join.
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:37]:
```suggestion
If a participant leaves their breakout room, the organizer can move the participant back to the breakout room by selecting the ellipses **[…]** next to the participant’s name and select **Ask to join**.
```

File: Teams/using-breakout-rooms.md [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+
+If a participant joins the main meeting late after breakout rooms have opened, the organizer can add the participant to a breakout room. If the participant misses the notification to join the room, select Ask to join.
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:39]:
```suggestion
If a participant joins the main meeting late after breakout rooms have opened, the organizer can add the participant to a breakout room. If the participant misses the notification to join the room, select **Ask to join**.
```

File: Teams/using-breakout-rooms.md [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+
+## Send Announcements
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:41]:
```suggestion
## Send announcements
```

File: Teams/using-breakout-rooms.md [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+
+The meeting organizer can send announcements to all participants while breakout rooms are open. Select the ellipses […] by the Breakout Rooms header.
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:43]:
```suggestion
The meeting organizer can send announcements to all participants while breakout rooms are open. Select the ellipses **[…]** in the breakout rooms header.
```

File: Teams/using-breakout-rooms.md [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+
+For example, remind participants to @mention the meeting organizer when they need help, or give them a fiveminute countdown before pulling them back into the main room.
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:45]:
```suggestion
For example, remind participants to @mention the meeting organizer when they need help, or give them a five-minute countdown before pulling them back into the main room.
```

File: Teams/using-breakout-rooms.md [hunk 16]
```diff
@@ -0,0 +1,53 @@
...
+Participants receive the announcement in their breakout rooms’ chat.
+
+## Close Rooms and Pull participants back to the main meeting
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:48]:
```suggestion
## Close rooms and pull participants back to the main meeting
```

File: Teams/using-breakout-rooms.md [hunk 17]
```diff
@@ -0,0 +1,53 @@
...
+
+Select Close rooms to pull all the participants back to the main room.
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:50]:
```suggestion
Select **Close rooms** to pull all the participants back to the main room.
```

File: Teams/using-breakout-rooms.md [hunk 18]
```diff
@@ -0,0 +1,53 @@
...
+
+Or select the ellipses […] for a specific breakout room, then select Close room.
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:52]:
```suggestion
Alternatively, select the ellipses **[…]** for a specific breakout room, then select **Close room**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Teams/using-breakout-rooms.md [hunk 19]
```diff
@@ -0,0 +1,53 @@
...
+
+4. Select **Create Rooms**. The breakout rooms pane will open.
+
+## Start breakout rooms
+
+When you are ready to start your breakout sessions, select **Start rooms**.
+
+You will see participants leaving the main meeting and joining their breakout rooms.
+
+If a participant leaves their breakout room, the organizer can move the participant back to the breakout room by selecting the ellipses **[…]** next to the participant’s name and select **Ask to join**.
```
Comment (JohanFreelancer9) [inline Teams/using-breakout-rooms.md:37]:
```suggestion
If a participant leaves their breakout room, the organizer can move the participant back to the breakout room by selecting the ellipses **[…]** next to the participant’s name and then selecting **Ask to join**.
```

**Pull request merged**